cds_sorenson/utils.py

```diff
@@ -101,8 +103,9 @@ def get_status(job_id):
 
 def _get_preset_config(preset_id):
     """Return preset config based on the preset_id."""
-    PRESETS = current_app.config['CDS_SORENSON_PRESETS']
-    presets_list = [item for presets in PRESETS.values() for item in presets]
-    for preset in presets_list:
-        if preset_id == preset.get('preset_id'):
-            return preset
+    return [
```

Comment:
Ok, disregarding the confusing names, that should be changed, I don't immediately see what is going on with this nested list comprehension. I know that list comprehension is widely accepted in Python community, but in this case it actually makes it less readable.
If I would come back to this code in 3 months and have to change it because the config file has changed, I would have to rewrite this completely, while when using a temporary variables, I can easily see (if not, I can as least step line by line with the debugger) which part has to be modified.
Everyone has personal preferences about code they write and we can argue, but in my opinion, we should write code that can be easily understood by people less-familiar with Python, that will have to maintain our code in the future.